src/cljs/main/broadfcui/main.cljs

```diff
@@ -120,68 +120,106 @@
+                       :backgroundColor background-color :padding "1rem"}}
+         [:div {:style {:display "flex" :alignItems "center" :marginBottom "0.5rem"}}
+          [icons/ExceptionIcon {:size 18 :color text-color}]
+          [:span {:style {:marginLeft "0.5rem" :fontWeight "bold"
+                          :verticalAlign "middle"}}
+           (if title title "Service Alert")]]
+         [:div {:style {:color text-color :fontSize "90%"}}
+          (str message " ")
+          (if more-content
+            [:div {} [:a {:href  "javascript:;" :onClick #(swap! state assoc :showing-more? (not (:showing-more? @state)))
+                          :style {:color "#000"}} (if (:showing-more? @state) "Hide details..." "Show details...")]])
+          (when (:showing-more? @state) more-content)
+          (when link link)]])])})
+
+(react/defc BannerContainer
   {:get-initial-state
    (fn []
-     {:failedRetries 0})
+     {:failed-retries 0})
    :render
    (fn [{:keys [this state]}]
-     (when-let [alerts (not-empty (:alerts @state))]
-       [:div {}
+     (let [service-alerts (:service-alerts @state) js-alerts (:js-alerts @state)]
+       ; Why was zIndex of 514 chosen? IGV has a zIndex of 512, so modals use a zIndex of 513
+       ; We want these banners to be shown in front of the modals, so we use a zIndex of 514
+       [:div {:style {:zIndex 514 :position "relative"}}
+        (map (fn [alert]
+               [Banner {:title (:title alert)
+                        :background-color (:exception-state style/colors)
+                        :text-color "#fff"
+                        :message (:message alert)
+                        :link [:a {:href (str (:link alert)) :target "_blank" :style {:color "#fff"}} "Read more..."]}])
+             service-alerts)
         (map (fn [alert]
-               (status-alert (:title alert) (:message alert) (:link alert)))
-             alerts)]))
+               [Banner {:title "Uh oh! Something may be wrong..."
+                        :background-color (:warning-state style/colors)
+                        :text-color "#000"
+                        :message "There was an error in FireCloud. It may not mean anything, but you should consider reloading the page to be safe."
+                        :more-content [:div {:style {:paddingTop "0.5rem"}}
+                                       [:div {} "Here are some details about the error that occurred. If you post this on our "
+                                        [:a {:href "http://gatkforums.broadinstitute.org/firecloud/categories/ask-the-firecloud-team"
+                                             :target "_blank" :style {}} "forum"] ", our team can take a look."]
+                                       [:div {:style {:paddingTop "0.5rem"}} [:div {:style {:fontWeight "bold"}} "Stack trace: "]]
+                                       (map (fn [line]
+                                              [:div {} line])
```

Comment:
yeah so i struggled with this a bit because you'd think if the lines were already split then it'd just work without any extra effort......but instead you get this:

<img width="1610" alt="screen shot 2017-04-28 at 3 23 51 pm" src="https://cloud.githubusercontent.com/assets/7257391/25544064/dbe538d4-2c26-11e7-846b-0177c392f436.png">

is there a way to stop it from ignoring the new lines that are already present?

love the style change- will include.